README.md

```diff
@@ -139,24 +139,29 @@ Finally, another great way to interact with the community, is to attend the mont
 ### Need help?
 Email [fdc3@finos.org](mailto:fdc3@finos.org) if you need help getting started in the FDC3 Community. If you encounter technical difficulties accessing repositories, joining Slack, mailing lists or meetings, please email [help@finos.org](mailto:help@finos.org).
 
-## Become an FDC3 Participant, Editor or Maintainer
-Email [fdc3@finos.org](mailto:fdc3@finos.org) if you would like to become an FDC3 Participant, Editor or Maintainer. For more details on FDC3's governance please see our [Governance document](GOVERNANCE.md).
+## Why should you get involved in FDC3?
+If you or your firm intends to make use of the FDC3 standard (by implementing a Desktop Agent or App Directory, by adding support to apps to interoperate  with others via FDC3, or even by using apps, Desktop Agents or App Directories written by others) then participating in the governance, maintenance and onward development of FDC3 Standard will help to protect and strengthen the ecosystem developing around FDC3. Doing so will also empower you to help guide the Standard in directions that are relevant to your use or that of your firm.
```

Comment:
```suggestion
If you or your firm intends to make use of the FDC3 standard (by implementing a Desktop Agent or App Directory, by adding support to apps to interoperate with others via FDC3, or even by using apps, Desktop Agents or App Directories written by others) then participating in the governance, maintenance and onward development of the FDC3 Standard will help to protect and strengthen the ecosystem developing around FDC3. Doing so will also empower you to help guide the Standard in directions that are relevant to your use or that of your firm.
```

Reply:
Some mixed capitalization of 'standard' in this paragraph. In the rest of this PR 'standard' is lower case, unless it appears in 'Standards Working Group'.